backend/bitboxbase/bitboxbase.go

```diff
@@ -599,6 +599,11 @@ func (base *BitBoxBase) SetLoginPassword(password string) error {
 	if !reply.Success {
 		return &reply
 	}
+	// FIXME: Change backend node management to status: offline
+	err = base.Deregister()
```

Comment:
Why are you deregistering here?

Reply:
Yep, good catch, that shouldn't be there :) :100:

Reply:
We should have just checked the code :P

Reply:
What is the FIXME for?

Reply:
That one also belongs to the `RebootBase()` method. 
I will remove it altogether here. 
Part of what I'm working on for the backend base management, auto-reconnects and different statuses based on what kind of restart/disconnect/shutdown happens.
This was a reminder to change the `Deregister()` to something else when restarting.